README.md

```diff
@@ -1 +1,50 @@
-# node-modules-core-lab
\ No newline at end of file
+# Importing fs, url and path
+
+## Objectives
+
+1. Write a script to import fs, url and path
+1. Invoke methods from core modules fs, url and path
```

Comment:
I think we can combine these first two objectives: "Write a script that uses methods from the core modules `fs`, `url`, and `path`."